eduvmstorebackend/eduvmstore/db/models.py

```diff
@@ -1,60 +1,59 @@
-from sqlalchemy import Column, Integer, String, DateTime, Float, Boolean, ForeignKey
-from .session import Base
-from datetime import datetime
+import uuid
+from email.policy import default
```

Comment:
@rickertmar, bist du dir sicher, dass der import für das default stimmt, wirkt auf jeden Fall etwas komisch für mich

Reply:
hab gerade in pycharm geschaut, der import wird auch gar nicht verwendet, kann denk ich raus. Mache ich aber auch sowieso in dem commit von mir.